core/src/main/java/hudson/diagnosis/HudsonHomeDiskUsageChecker.java

```diff
@@ -66,6 +67,7 @@ protected void doRun() {
     /**
      * Gets the minimum amount of space to check for, with a default of 10GB
      */
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
     public static long FREE_SPACE_THRESHOLD = Long.getLong(
```

Comment:
This should access `SystemProperties` for the value (unless that doesn't support longs?)